client/src/features/users/Auth/Steps/CreateOrgForm.tsx

```diff
@@ -0,0 +1,285 @@
+import { Button, FormHelperText, Grid, LinearProgress, Typography } from '@material-ui/core';
+import CheckIcon from '@material-ui/icons/Check';
+
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import React from 'react';
+import { Select, TextField } from '../../../../assets/sharedComponents/Forms';
+import { Organization } from '../../../../types';
+import { ClassNameMap } from '@material-ui/core/styles/withStyles';
+import { useMutation, useQuery } from 'react-query';
+import axios, { AxiosResponse } from 'axios';
+import SimpleSnackbar from '../../../action/assets/SimpleSnackbar';
+import { green } from '@material-ui/core/colors';
+
+const classifications = [
+  { value: 'charitable', text: 'Charitable Organization' },
+  { value: 'religious', text: 'Religious Organization' },
+  { value: 'private', text: 'Private Foundation' },
+  { value: 'political', text: 'Political Organizations' },
+  { value: 'other', text: 'Other' },
+];
+
+const SignupSchema = Yup.object().shape({
+  doing_business_as: Yup.string().required('Required'),
+  city: Yup.string().required('Required'),
+  state: Yup.string().required('Required').min(2).max(2),
+  ein: Yup.string()
+    .matches(/^[0-9]\d?-\d{7}$/, 'EIN must match: 99-9999999')
+    .required('Required'),
+  name: Yup.string(),
+  description: Yup.string().required('Required'),
+  website: Yup.string().required('Required'),
+  address: Yup.string().required('Required'),
+  phone: Yup.string()
+    .matches(/^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]\d{3}[\s.-]\d{4}$/, 'EIN must match: 99-9999999')
+    .required('Required'),
+  nonprofit_classification: Yup.string().min(2).required('Required'),
+});
+
+interface Props {
+  triggerNextStep: (step: number) => void;
+  setParentOrg: (createdOrg: Organization) => void;
+  parentOrg: Organization;
+  classes: ClassNameMap<any>;
+}
+const CreateOrgForm = ({
+  setParentOrg = () => {},
+  parentOrg,
+  triggerNextStep = () => {},
+  classes,
+}: Props) => {
+  const [org, setOrg] = React.useState<Organization>(parentOrg);
+  const [triggerEinSearch, setTriggerEinSearch] = React.useState<boolean>(false);
+
+  const orgCreateMutation = useMutation<AxiosResponse<any, any>, Error, Organization, Error>(
+    (newOrg: Organization) => {
+      return axios.post(`http://localhost:3001/api/organizations`, { ...newOrg });
+    },
+    {
+      onSuccess: (data: any) => {
+        triggerNextStep(1);
+        setParentOrg((data as any).data);
+      },
+    },
+  );
+
+  const orgValidateEinQuery = useQuery<
+    AxiosResponse<any, any>,
+    unknown,
+    { ein: string; name: string },
+    string[]
+  >({
+    enabled: triggerEinSearch,
+    queryKey: ['orgValidateEinQuery', org.ein],
+    queryFn: ({ queryKey }) => {
+      const [_key, ein] = queryKey;
+      console.log(_key); // need to use _key somehow
```

Comment:
i think you can destructure like `[, ein] =`

Reply:
Nice. It worked